README.md

```diff
@@ -29,7 +29,10 @@ ember install ember-test-waiters
 
 ### buildWaiter function
 
-The `buildWaiter` function is, in most cases, all you will need to wait for async operations to complete before continuing tests.
+The `buildWaiter` function is, in most cases, all you will need to wait for async operations to complete before continuing tests. It returns a waiter instance
```

Comment:
`in most cases` are there cases u have in mind where this won't cover? should it be called out?

Reply:
Well yes, it would be any case that you would want to define your own custom waiter class. I believe that 99 times out of 100 you’ll just want to use buildWaiter.